### ISSUE TITLE
[NOT Bug]: Feature requests, TranslucentTB modifies taskbar according to theme of the desktop wallpaper

### ISSUE THREAD

**Issue by MMMMMoris** [TranslucentTB/TranslucentTB#963]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

I usually use dark mode on my PC with dynamic desktop wallpaper. Sometimes the wallpaper is dark theme mainly, while it is light theme mainly and the taskbar is clear status, everything becomes difficult to recognize, icons, letters. So, I wish there is a feature that the program can automatically change the taskbar status according to the theme of the desktop wallpaper, so that users can see the things on taskbar clearly. If the wallpaper is mainly dark, TranslucentTB makes the taskbar whatever clear or acrylic, user can also define this. And when the wallpaper turns light, TranslucentTB makes the taskbar the status user set it to be. 

### Expected results

TranslucentTB modifies taskbar according to theme of the desktop wallpaper

### Actual results

TranslucentTB cannot modify taskbar according to theme of the desktop wallpaper

### Windows version

Windows 11 SV2 (builds 22621-22622)

### Other details

Hope it can be done in the future

**Comment by kanxingyv** [comment]
I'm also facing this issue. Hope it gets resolved soon.

**Comment by carlosvsilva** [comment]
Yeah, same here as I use AutoDarkMode to have Windows on dark mode only at night to be easier on the eyes. I hope TranslucentTB can enable soon Light and Dark Mode support of Windows 11.

Right now I use "Clear" with a specific tint in daytime, and "Normal" at night time, but have to change manually...

Thanks.

**Comment by tmprnc** [comment]
I am in the same case as the user above. I use AutoDarkMode with two different themes. I'd like to have TranslucentTB settings change depending on which theme I'm using.
